docs/showcases/graphql.md

```diff
@@ -0,0 +1,135 @@
+# Legend GraphQL Showcase  
+
+This showcase demonstrates the uses of the GraphQL protocol in Legend. 
+
+## Showcase requirements 
+
+* An installation of Legend with the following components :
+    * Studio 
+    * Engine 
+    * Depot Store 
+    * Depot 
+    * GraphiQL client
+  
+* Legend Studio Project - https://gitlab.com/finosfoundation/legend/showcase/legend-graphql-showcase 
+
+## Showcase setup 
+
+__1. Install Legend__
```

Comment:
maybe let's standardize and use deeper-level header `###` instead?

Reply:
Done